### ISSUE TITLE
The model and loaded state dict do not match exactly

### ISSUE THREAD

**Issue by 1428679766** [open-mmlab/mmdetection#3915]
When loading some pre-training backbones, there are warnings like "missing keys in source", how can I fix this? Does it affect the accuracy of the final training model?

**Comment by v-qjqs** [comment]
Hi @1428679766, it's common that there exists missing keys or unexpected keys in the source state_dict. For example, there exists unexpected key in source state_dict like 'fc.weight', since the classification head is not used in the detector. And there may exists missing keys (e.g. DCN's weights) if new module is added to the pretrained backbone.
